README.md

```diff
@@ -22,163 +23,70 @@
 
 # Overview
```

Comment:
As soon as this is merged into `master`, the README will update. For this PR I would suggest:

1. Keeping the same README as before and
2. Just removing the row in the table that says `LICENSE`, so that it's not conflicting with the changes

<img width="1040" alt="image" src="https://user-images.githubusercontent.com/3227804/208184811-af28fce2-8e77-472c-a9f9-61e0b94f40eb.png">